test/regression/cases/uds_dogstatsd_to_api/lading/lading.yaml

```diff
@@ -1,15 +1,37 @@
+# If you update this configuration please update `uds_dogstatsd_*` experiments
+# to match, where reasonable.
 generator:
   - unix_datagram:
       seed: [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53,
              59, 61, 67, 71, 73, 79, 83, 89, 97, 101, 103, 107, 109, 113, 127, 131]
       path: "/tmp/dsd.socket"
       variant:
         dogstatsd:
-          metric_names_minimum: 32
-          metric_names_maximum: 128
-          tag_keys_minimum: 0
-          tag_keys_maximum: 512
-      bytes_per_second: "100 Mb"
+          contexts_minimum: 1000
+          contexts_maximum: 10000
+          name_length_minimum: 1
+          name_length_maximum: 200
+          tag_key_length_minimum: 1
+          tag_key_length_maximum: 100
+          tag_value_length_minimum: 1
+          tag_value_length_maximum: 100
+          tags_per_msg_minimum: 2
+          tags_per_msg_maximum: 50
+          multivalue_count_minimum: 2
+          multivalue_count_maximum: 32
+          multivalue_pack_probability: 0.08
+          kind_weights:
+            metric: 100
+            event: 0
+            service_check: 0
```

Comment:
Could you add an experiment to cover these?

Reply:
Ah yes. I will amend the existing experiments to add these in.

Reply:
Addressed in [00c7869](https://github.com/DataDog/datadog-agent/pull/19324/commits/00c786932fbabe4f4ea3188fa31126e6f592e2f0)